codalab/lib/bundle_cli.py

```diff
@@ -1931,7 +1933,10 @@ def do_search_command(self, args):
 
         # Print table
         if len(bundles) > 0:
+            print('Note: at most {} results will be shown.'.format(bundle_model.SEARCH_RESULTS_LIMIT))
```

Comment:
Only print this out when `len(bundles) == SEARCH_RESULTS_LIMIT`.  Otherwise, it's the full list.

Reply:
And in that case, print "Only {} results are shown.  Use .limit=N to show N."

Reply:
Sounds good.